help/bug_report.md

```diff
@@ -4,13 +4,20 @@ title: Bug Reports
 permalink: /bug_report/
 ---
 
-* <a href="https://github.com/triplea-game/triplea/issues?q=is%3Aissue+is%3Aopen+label%3ABug">Open Bug List</a>
+* [Open Bug List](https://github.com/triplea-game/triplea/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A%20bug%22)
 
-## Filing a bug report:
+## Before doing anything
 
-* Create a <a href="https://github.com/account"> github account</a>
-* Submit a tracking issue: <a href="https://github.com/triplea-game/triplea/issues/new">https://github.com/triplea-game/triplea/issues/new</a>, you'll need to give the topic a title, and fill out a description of what you would like done.  Please try to be as concise and specific as you can. Always please try to give specific examples of problems, map names, etc, to help make it very easy to focus in directly on what the problem is. 
+Before submitting a bug report at all, take a step back and make sure it's really a bug and not a feature request.
```

Comment:
I recommend striking this first sentence. We would need to tell our users how to determine if something is existing or not, and it is all extra hurdles for something that needs to be easy (like stupid easy). The natural reaction of most users at just seeing this page and the amount of text is probably to close their browser window, let alone after seeing how much stuff they have to do.

I also think that de-duping bug reports is our jobs as admins, our bugs better not be so pervasive, otherwise we have a p0, or we have a pretty important update to make since we are getting so much feedback.